src/templates/auth_manifests.yaml.j2

```diff
@@ -0,0 +1,49 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
(this is my sidecar/pebble-ignorance showing) is there no sidecar-native way to grant these permissions?  I'd have expected this to be necessary for odd cases, but not for something standard like this.  @DomFleischmann @DnPlas ?

Reply:
I'd expect the charm to have permissions granted for most standard k8s resources, but probably not the workload, and also I am not so sure those permissions are extended to custom resources.

Reply:
Without the cluster role I am not able to create the profile CRD (e.g. in integration test).

Reply:
Yeh so it is the latter case: permissions are not extended to the workload to manage custom resources